packages/itmat-interface/src/graphql/schema.ts

```diff
@@ -250,6 +250,13 @@ type Mutation {
     # USER
     login(username: String!, password: String!): User
     logout: GenericResponse
+    requestUsernameOrResetPassword(
```

Comment:
`requestUsernameOrPasswordReset` ?

Reply:
request

> In fact I would probably remove that, and use a token only. That avoid man-in-the-middle type of scenario.
How would removing the username avoid man-in-the-middle?